### ISSUE TITLE
x server blank

### ISSUE THREAD

**Issue by grantperry** [DamianB-BitFlipper/JS-OS#2]
this could be an issue to do with my computer and or qemu but when the x server is started the screen (resises in qemu) then goes blank

**Comment by ghost** [comment]
Try starting QEMU with `-vga std` and see if it works.

**Comment by grantperry** [comment]
still no luck with `-vga std` option

**Comment by DamianB-BitFlipper** [comment]
Yes, this part of JS-OS has been, easily put, not the most stable. When developing, I did not put that much emphasis on the graphics since it is not that crucial to an operating system in early development. It would work and then break, etc. What happens, if I can recall correctly, is that when initializing into VESA mode, I had to allocate a buffer of a decent size to represent the graphics buffer. That process caused a page fault and thus a system-wide crash.

**Comment by grantperry** [comment]
Thanks,
So are you still working on this project or is it dead?

**Comment by DamianB-BitFlipper** [comment]
I would not say this project is dead, it is suspended. I will be coming back to it later, the only reason I suspended it is because I am working on another project of mine which I want to see up and running,

**Comment by grantperry** [comment]
Good. it is a great resource for my project.
Thank You.
